internal/pkg/db/db.go

```diff
@@ -49,6 +49,8 @@ const (
 	DatabaseFile string = "bomctl.db"
 
 	EntDebugLevel int = 2
+
+	OriginalFormat = "original"
```

Comment:
In the call, what I was suggesting was

```suggestion
	OriginalFormat = formats.Format("original")
```

Will that work? Or does it still require casting to string

Reply:
It works the same as the initial version. meaning it requires casting to a string in `cmd/export.go`. I went this way because the `formats.SPDX` and the like are untyped strings in protobom, so this matches that pattern

Reply:
I'm going to keep it as an untyped string and remvoe the cast in your other comment. I think thats the best option